lib/boat_tracker/application.ex

```diff
@@ -13,7 +13,7 @@ defmodule BoatTracker.Application do
     opts = [strategy: :one_for_one, name: BoatTracker.Supervisor]
 
     # Children for all targets
-    children = [GPS | children(target())]
+    children = children(target())
```

Comment:
I think we can discard this change since it's already in the Parser PR

Reply:
Tests will fail if this change is not present. This change does not `init` the `UART` for hosts (everything except the RPi).